docs/getting_started/installation.rst

```diff
@@ -1,6 +1,26 @@
 Installation
 ============
 
-.. warning::
+Before installing Qililab it is recommended to create a virtual environment, to do it so
+please run:
+
+.. code-block:: console
+
+    $ python3 -m venv .venv
+
+and activate it like this:
+
+.. code-block:: console
+
+    $ source .venv/bin/activate
+
+To install Qililab, please run:
+
+.. code-block:: console
+
+    $ git clone https://github.com/qilimanjaro-tech/qililab
+    $ cd qililab
+    $ pip install -r requirements.txt
+
+This will install all necessary dependencies to start using Qililab.
 
```

Comment:
what about `pip install -e .` to install qililab?

Reply:
This is for general users, not developers. That step is already in the documentation for developers.